packages/cli/src/cache.js

```diff
@@ -17,8 +17,25 @@ export function getDefaultCachePath() {
   return path.join('node_modules', '.stylex-cache');
 }
 
+async function findProjectRoot(startDir = __dirname) {
+  let currentDir = path.resolve(startDir);
+  while (currentDir !== path.parse(currentDir).root) {
+    const packageJsonPath = path.join(currentDir, 'package.json');
+    try {
+      await fs.access(packageJsonPath);
```

Comment:
TODO:

Check for `deno.json` and other similar files that may exist. This is fine for now.